apps/finance/contracts/Finance.sol

```diff
@@ -105,23 +102,20 @@ contract Finance is AragonApp, ERC677Receiver {
      * @notice Allows to send ETH from this contract to Vault, to avoid locking them in contract forever.
      */
     function escapeHatch() public payable {
-        // convert ETH to EtherToken
-        etherToken.wrapAndCall.value(this.balance)(address(this), "Adding Funds");
+        vault.deposit.value(msg.value)(address(0), msg.sender, 0, new bytes(0));
```

Comment:
Value must be `msg.value`, this code should only work when `msg.value == 0`

https://github.com/aragon/aragon-apps/pull/129/files#diff-9ebb8e9a2c4b877825d0aeece0fb63feR14